### DISCUSSION TITLE
how can i install hds with linux terminal?

### DISCUSSION THREAD

**Discussion by jslee93** [Rexios80/hds_overlay#59] [Q&A]
I had installed HDS from a previous version.

  When I typed "./HDS-Overlay-linux" I could see the HDS data being pinged from the terminal.

Since it has stopped working for some time, I am trying to install a new version. But I don't know how.

**Comment by Rexios80** [comment]
I'm gonna be honest I have no idea if the Linux version of the new overlay works. It should just be a normal program you open by double clicking. You might have to chmod it first but I'm not sure.

Also the desktop apps are kind of useless now unless you want more than just heart rate and don't want to pay for HDS Cloud.

**Comment by Rexios80** [comment]
Did you get it figured out?
